src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -435,13 +469,39 @@ subroutine diabatic(u, v, h, tv, Hml, fluxes, visc, ADp, CDp, dt, Time_end, &
 
   if (CS%debugConservation) call MOM_state_stats('leaving diabatic', u, v, h, tv%T, tv%S, G, GV, US)
 
+#ifdef UFS
```

Comment:
Is the if def necessary here given the "do_sppt" conditional?

Reply:
It would not be when I remove the ifdef at line 297